youtube_dl/options.py

```diff
@@ -340,20 +340,17 @@ def _hide_login_info(opts):
         action='store_true', dest='listformats',
         help='List all available formats of requested videos')
     video_format.add_option(
-        '--youtube-include-dash-manifest',
-        action='store_true', dest='youtube_include_dash_manifest', default=True,
-        help=optparse.SUPPRESS_HELP)
-    video_format.add_option(
-        '--youtube-skip-dash-manifest',
-        action='store_false', dest='youtube_include_dash_manifest',
-        help='Do not download the DASH manifests and related data on YouTube videos')
-    video_format.add_option(
         '--merge-output-format',
         action='store', dest='merge_output_format', metavar='FORMAT', default=None,
         help=(
             'If a merge is required (e.g. bestvideo+bestaudio), '
             'output to given container format. One of mkv, mp4, ogg, webm, flv. '
             'Ignored if no merge is required'))
+    video_format.add_option(
+        '--skip-protocols',
```

Comment:
if we try to add other types of steps i think we should update extractor to use common methods so it will be simpler to add this contraits like.
_real_extract_media_formats, _real_extract_subtitles, _real_extract_metadata... that extractors will override and wrap them inside methods without _real where we can check for the --skip-extraction-steps and may add other behavior(messages about starting the specific step)